demo/package.json

```diff
@@ -16,14 +17,15 @@
     "@babel/preset-react": "^7.0.0-beta.55",
     "@babel/preset-typescript": "^7.3.3",
     "babel-loader": "^8.0.0-beta.0",
-    "css-loader": "2.1.1",
-    "html-webpack-plugin": "3.2.0",
-    "node-sass": "^4.12.0",
-    "sass-loader": "7.1.0",
-    "style-loader": "0.23.1",
-    "webpack": "4.33.0",
-    "webpack-cli": "^3.3.4",
-    "webpack-dev-server": "^3.7.1"
+    "patch-package": "^6.4.7",
+    "css-loader": "6.7.1",
+    "html-webpack-plugin": "5.5.0",
+    "node-sass": "7.0.1",
```

Comment:
What if switched to `sass` instead of `node-sass`? It looks like that's the [recommended way](https://github.com/sass/node-sass#node-sass) to use sass these days. With that I assume we won't need to patch packages anymore and the `sass` package will work out of the box with newer node versions.

Reply:
sass works good. And also eliminate the cpp toolchain. Should be more friendly for windows user now.